tests/unit_tests/pyanaconda_tests/modules/payloads/source/test_source_rpm_ostree_container.py

```diff
@@ -0,0 +1,138 @@
+#
+# Copyright (C) 2023  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+import unittest
+
+from dasbus.typing import get_variant, Str, Bool
+
+from pyanaconda.core.constants import SOURCE_TYPE_RPM_OSTREE_CONTAINER
+from pyanaconda.modules.common.constants.interfaces import PAYLOAD_SOURCE_RPM_OSTREE_CONTAINER
+from pyanaconda.modules.payloads.constants import SourceType, SourceState
+from pyanaconda.modules.common.structures.rpm_ostree import RPMOSTreeContainerConfigurationData
+from pyanaconda.modules.payloads.source.rpm_ostree_container.rpm_ostree_container import \
+    RPMOSTreeContainerSourceModule
+from pyanaconda.modules.payloads.source.rpm_ostree_container.rpm_ostree_container_interface \
+    import RPMOSTreeContainerSourceInterface
+
+from tests.unit_tests.pyanaconda_tests import check_dbus_property
+
+
+class OSTreeContainerSourceInterfaceTestCase(unittest.TestCase):
+    """Test the DBus interface of the OSTree source."""
+
+    def setUp(self):
+        self.module = RPMOSTreeContainerSourceModule()
+        self.interface = RPMOSTreeContainerSourceInterface(self.module)
+
+    def _check_dbus_property(self, *args, **kwargs):
+        check_dbus_property(
+            PAYLOAD_SOURCE_RPM_OSTREE_CONTAINER,
+            self.interface,
+            *args, **kwargs
+        )
+
+    def test_type(self):
+        """Test the Type property."""
+        assert SOURCE_TYPE_RPM_OSTREE_CONTAINER == self.interface.Type
+
+    def test_description(self):
+        """Test the Description property."""
+        assert "RPM OSTree Container" == self.interface.Description
+
+    def test_configuration(self):
+        """Test the configuration property."""
+        data = RPMOSTreeContainerConfigurationData()
+        data.osname = "fedora-coreos"
+        data.remote = "fcos-28"
+        data.url = "quay.io/fedora/coreos:stable"
+        data._signature_verification_enabled = False
```

Comment:
BTW, the FCOS image actually does have ostree-based GPG signatures, and verifying those *does* work.

Reply:
These are just test values, we don't run any ostree command here.